api/drizzle/seeder/assign-images.ts

```diff
@@ -0,0 +1,29 @@
+import { eq } from "drizzle-orm"
+
+import { db } from "../../src/utils/db"
+import { products as productSchema } from "../../src/schema/products"
+
+const SHOPS = ["stitch", "luminara", "pug"]
+
+/**
+ * Assign images to products based on their shops category.
+ */
+export async function assignImagesToProducts() {
```

Comment:
One-time script used to assign images. Likely will never need to be ran again, but I added it here solely for future reference.